libraries/plugins/peerplays_sidechain/sidechain_net_handler.cpp

```diff
@@ -68,7 +68,7 @@ void sidechain_net_handler::sidechain_event_data_received(const sidechain_event_
    const chain::global_property_object &gpo = database.get_global_properties();
 
    // Deposit request
-   if ((sed.peerplays_to == gpo.parameters.son_account()) && (sed.sidechain_currency.compare("1.3.0") != 0)) {
+   if ((sed.peerplays_to == gpo.parameters.son_account()) && (sed.sidechain == sidechain_type::bitcoin)) {
```

Comment:
You have to track exact asset here, not whole sidechain, because deposit can be done in any peerplays asset except BTC asset.